### PR TITLE
HPCC-21614 ECL Watch is not rendering inside the IDE

### PR DESCRIPTION
**Pull request by GordonSmith**
Signed-off-by: Gordon Smith <gordonjsmith@gmail.com>

## Type of change:
- [x] This change is a bug fix (non-breaking change which fixes an issue).
- [ ] This change is a new feature (non-breaking change which adds functionality).
- [ ] This change improves the code (refactor or other change that does not change the functionality)
- [ ] This change fixes warnings (the fix does not alter the functionality or the generated code)
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).
- [ ] This change alters the query API (existing queries will have to be recompiled)

## Checklist:
- [x] My code follows the code style of this project.
  - [ ] My code does not create any new warnings from compiler, build system, or lint.
- [x] The commit message is properly formatted and free of typos.
  - [x] The commit message title makes sense in a changelog, by itself.
  - [x] The commit is signed.
- [ ] My change requires a change to the documentation.
  - [ ] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [x] I have read the CONTRIBUTORS document.
- [x] The change has been fully tested:
  - [ ] I have added tests to cover my changes.
  - [ ] All new and existing tests passed.
  - [ ] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [x] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [ ] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Premature optimization
  - [ ] Existing deployed queries will not be broken
  - [ ] This change fixes the problem, not just the symptom
  - [ ] The target branch of this pull request is appropriate for such a change.
- [x] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised JIRA issues to address them separately
- [x] This is a user interface / front-end modification
  - [x] I have tested my changes in multiple modern browsers
  - [x] The component(s) render as expected

### CODE REVIEW

**Comment by GordonSmith** [comment]
@miguelvazq - I was not able to fully test with ldap enabled.

File: esp/src/eclwatch/Graph7Widget.js [hunk 1]
```diff
@@ -1,24 +1,27 @@
...
 
     "@hpcc-js/eclwatch",
 
-    "dojo/text!../templates/Graph7Widget.html"
-], function (declare, lang, i18n, nlsHPCC, dom, domClass, domStyle, Evented,
-    registry, BorderContainer, ContentPane,
+    "dojo/text!../templates/Graph7Widget.html",
+
+    "dijit/layout/BorderContainer",
+    "dijit/layout/ContentPane",
```
Comment (miguelvazq) [inline esp/src/eclwatch/Graph7Widget.js]: Not used

File: esp/src/eclwatch/Graph7Widget.js [hunk 2]
```diff
@@ -1,24 +1,27 @@
...
+    "dijit/ToolbarSeparator",
```
Comment (miguelvazq) [inline esp/src/eclwatch/Graph7Widget.js]: Not used

**Review by miguelvazq — COMMENTED** [review]
Just some minor comments otherwise code looks good.

**Comment by miguelvazq** [comment]
@kunalaswani Can you verify the testing

**Review by kunalaswani — APPROVED** [review]
code looks good

File: esp/src/eclwatch/Graph7Widget.js (hunk 1 above)
Comment (GordonSmith) [reply esp/src/eclwatch/Graph7Widget.js]: I think they are (in dojo/text!../templates/Graph7Widget.html)?

**Comment by miguelvazq** [comment]
Looks good.

File: esp/src/eclwatch/Graph7Widget.js (hunk 1 above)
Comment (miguelvazq) [reply esp/src/eclwatch/Graph7Widget.js]: Indeed they are.

**Review by miguelvazq — APPROVED** [review]

**Comment by GordonSmith** [comment]
This needs to be tested on a LDAP enabled server...

**Comment by kunalaswani** [comment]
I did encounter a pop up message for a "script error", upon resubmitting that datafile though, eclwatch renders as expected.

**Comment by kunalaswani** [comment]
(When selecting ECLWatch tab after submission)
Info From Scirpt Error:
An error has occured in the script on this page.
Line: 16432
Char: 16
Error: Expected Identifier
Code: 0
URL: ip:8010/esp/files/dist/stub.eclwatch.js

**Comment by GordonSmith** [comment]
@kunalaswani can you fetch the latest version of this PR and retest - I believe it to be working now...

**Comment by kunalaswani** [comment]
Built and tested, no longer seeing "script error" and after logging into eclwatch in the ide it loads eclwatch.

**Pull request merged**